style.css

```diff
@@ -0,0 +1,772 @@
+}
+
+nav .nav-menu {
+  display: none;
+  position: fixed;
+  top: 50px;
+  left: 0;
+  height: calc(100vh - 50px);
+  width: 100%;
+}
+
+.open .nav-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  list-style-type: none;
+  background-color: rgba(211, 211, 211, 0.1);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  z-index: 1;
+  padding: 30px 30px;
+}
+
+.nav-menu li {
+  margin-bottom: 40px;
+  cursor: pointer;
+}
+
+.nav-menu li a {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+}
+
+.open .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-6px, 6px);
+  transform: rotate(-45deg) translate(-6px, 6px);
+}
+
+.open .bar2 {
+  opacity: 0;
+}
+
+.open .bar3 {
+  -webkit-transform: rotate(45deg) translate(-6px, -8px);
+  transform: rotate(45deg) translate(-6px, -8px);
+}
+
+.links-container {
+  display: none;
+}
+
+header {
+  padding: 70px 25px 25px 50px;
+}
+
+header .header-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 35px;
+}
+
+.header-container .welcome-heading {
+  color: #ec5242;
+  font-family: "Lato", sans-serif;
+  font-size: 17px;
+}
+
+.header-container .intro {
+  font-family: "cocogoose-trial", sans-serif;
+  text-transform: uppercase;
+  display: block;
+  margin-top: 30px;
+  color: rgba(236, 82, 66, 1);
+  -webkit-background-clip: text;
+  font-size: 40px;
+  max-width: 500px;
+  font-weight: normal;
+}
+
+.header-container .intro-paragraph {
+  border: 2px solid white;
+  background-color: #d3d3d3;
+  padding: 15px;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #4a4848;
+  margin-top: 30px;
+  max-width: 500px;
+}
+
+.header-container .date {
+  font-family: "Cocogoose-trial", sans-serif;
+  font-size: 30px;
+  color: #4a4848;
+  margin-top: 25px;
+}
+
+.main-program {
+  background: url(assets/mailing-bg.jpg);
+  padding-top: 25px;
+  padding-bottom: 25px;
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+}
+
+.main-program .main-program-heading {
+  color: white;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  margin-bottom: 12px;
+}
+
+.main-program-hr {
+  width: 35px;
+  align-self: center;
+  height: 1px;
+  background-color: #ec5242;
+}
+
+.main-program .programs-list {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  padding-right: 25px;
+  padding-left: 25px;
+  margin-top: 25px;
+  align-self: center;
+}
+
+.main-program .programs-list .program-container {
+  display: flex;
+  flex-direction: row;
+  background: rgba(238, 233, 228, 0.1);
+  border-bottom: 40px;
+  margin-bottom: 10px;
+  padding: 10px 10px 10px 10px;
+  max-width: 350px;
+}
+
+footer .footer-logo-container img {
+  width: 135px;
+}
+
+.featured-professors .more-professors-btn img {
+  margin-left: 25px;
+}
+
+.past-summits-images-container .img-container img {
+  width: 300px;
+  filter: sepia(100%);
+  -webkit-filter: sepia(100%);
+}
```

Comment:
- [ ] As suggested by the previous reviewer about the feature speakers alignment. You made a little adjustment, but does not match the guiding template yet. What I noticed is that you need to have a parent container that would determine the margin at the both sides. With you would have removed many problem of adjusting each section on the page. You need to give margin left and right to the parent container and every other sections in the parent container would fall within the container. Kindly look deeply into this and fix it. Take a look. :point_down: 

### Your Design
[your-page-overall.webm](https://user-images.githubusercontent.com/26597432/225893896-18c3cac3-fd3f-48bb-9d75-de0277709550.webm)

### Guiding Design
[temp-overall-page.webm](https://user-images.githubusercontent.com/26597432/225893969-1e169e63-d32a-4952-8a33-c48c4c20ca14.webm)

Reply:
@Qoosim Pictures are not clear here

Reply:
Hi @Qoosim, Can you please re-upload the pictures in this section as they are not showing anything? I cannot see what changes I should make here.

Reply:
It's ok it's approved already. Thank you anyways. "Hope your prayers is accepted, and soon you would pray in Al-Haram" 🕋

Reply:
@MarwanDev, Aamin masha Allah. Thanks for the prayer. :palms_up_together: